node/_fs/_fs_streams_test.ts

```diff
@@ -0,0 +1,203 @@
+// Copyright 2018-2022 the Deno authors. All rights reserved. MIT license.
+import { createReadStream, ReadStream } from "./_fs_streams.ts";
+import * as path from "../../path/mod.ts";
+import { assertEquals } from "../../testing/asserts.ts";
+
+const moduleDir = path.dirname(path.fromFileUrl(import.meta.url));
+const testData = path.resolve(moduleDir, "testdata", "hello.txt");
+
+Deno.test({
```

Comment:
Instead of adding a bunch of tests in this PR, is it possible to import more tests from the Node test suite? Ideally, we should rely on the Node test suite for Node compat and add tests for the things that the Node test suite might not cover.

Reply:
Sure. Please give me some time to see which Node test suite can be imported.

Reply:
Duplicate Unit test cases have been reorganized in https://github.com/denoland/deno_std/pull/2634/commits/67266461986bf0938bb03c18a5a777e1ab010b49 and https://github.com/denoland/deno_std/pull/2634/commits/a7a852cdcaa8adc9a34312ea3c8df7e2e752f464.
Regarding the Node test suite, please see the separate comment added below.

Reply:
Updated the comment regarding the Node test suite. https://github.com/denoland/deno_std/pull/2634#issuecomment-1241522250